test/hotspot/jtreg/ProblemList.txt

```diff
@@ -110,7 +110,7 @@ resourcehogs/serviceability/sa/TestHeapDumpForLargeArray.java 8262386 generic-al
 serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatIntervalTest.java 8214032 generic-all
 serviceability/jvmti/HeapMonitor/MyPackage/HeapMonitorStatArrayCorrectnessTest.java 8224150 generic-all
 serviceability/jvmti/ModuleAwareAgents/ThreadStart/MAAThreadStart.java 8225354 windows-all
-serviceability/dcmd/gc/RunFinalizationTest.java 8227120 linux-x64
+serviceability/dcmd/gc/RunFinalizationTest.java 8227120 linux-all,windows-x64
```

Comment:
Should linux-all be linux-aarch64?

Reply:
No, the entry was previously linux-x64 so I've changed it to linux-all to
cover linux-x64 and linux-aarch64.

Reply:
Looks good and trivial then.